protocol/src/main/java/org/geysermc/mcprotocollib/protocol/data/game/chunk/DataPalette.java

```diff
@@ -10,26 +10,13 @@
 @ToString
 public class DataPalette {
 
-    /*
-     * @deprecated globalPaletteBits is no longer in use.
-     */
-    @Deprecated(forRemoval = true)
     public static final int GLOBAL_PALETTE_BITS_PER_ENTRY = 14;
-
+    private final PaletteType paletteType;
     private @NonNull Palette palette;
     private BitStorage storage;
-    private final PaletteType paletteType;
-
-    /*
-     * @deprecated globalPaletteBits is no longer in use, use {@link #DataPalette(Palette, BitStorage, PaletteType)} instead.
-     */
-    @Deprecated(forRemoval = true)
-    public DataPalette(@NonNull Palette palette, BitStorage storage, PaletteType paletteType, int globalPaletteBits) {
-        this(palette, storage, paletteType);
-    }
```

Comment:
Why did you remove the deprecation comments here?

Reply:
It's forRemoval.

Reply:
dev already requires a major release as package names changed

Reply:
It's still in use, the javadoc is wrong.

Reply:
Where is it currently used except in methods that will also be removed?

Reply:
sanitizeBitsPerEntry(int)